src/frontend/helpers/achievementUtils.ts

```diff
@@ -0,0 +1,6 @@
+export function isTimestampInPast(unixTimestamp: number) {
```

Comment:
This should go to date utilities since can be utilized in other places

Reply:
okay will move to common utils